packages/near-api-js/test/wallet-account.test.js

```diff
@@ -17,6 +17,7 @@ const nearApi = require('../src/index');
 
```

Comment:
Can we add tests for the assertions in https://github.com/near/near-api-js/pull/927/files/40e049b3ee19c00bd80d21c0890f84da2ab04d44#r953151759 by setting things in localStorage before constructing a `WalletConnection` instance?

Also do we have to reset localStorage before each test?